### PR TITLE
Update sensor.yr.markdown

### PR DESCRIPTION
**Pull request by Danielhiversen**
**Description:**
Add location parameter to config

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/sensor.yr.markdown [hunk 1]
```diff
@@ -68,6 +68,21 @@ monitored_conditions:
       description: Medium cloud level.
     highClouds:
       description: High cloud level.
+latitude:
+  description: Manually specify latitude. By default the value will be taken from the Home Assistant configuration.
```
Comment (DubhAd) [inline source/_components/sensor.yr.markdown]: By default the value will be taken from the Home Assistant configuration.

Seems redundant when you mention the default immediately below ;)

File: source/_components/sensor.yr.markdown [hunk 2]
```diff
@@ -68,6 +68,21 @@ monitored_conditions:
...
+  required: false
+  type: number
+  default: Provided by Home Assistant configuration
+longitude:
+  description: Manually specify longitude. By default the value will be taken from the Home Assistant configuration.
```
Comment (DubhAd) [inline source/_components/sensor.yr.markdown]: Same

File: source/_components/sensor.yr.markdown [hunk 3]
```diff
@@ -68,6 +68,21 @@ monitored_conditions:
...
+  required: false
+  type: number
+  default: Provided by Home Assistant configuration
+altitude:
+  description: Manually specify altitude. By default the value will be taken from the Home Assistant configuration.
```
Comment (DubhAd) [inline source/_components/sensor.yr.markdown]: Same

**Review by DubhAd — DISMISSED** [review]

File: source/_components/sensor.yr.markdown [hunk 4]
```diff
@@ -68,6 +68,21 @@ monitored_conditions:
       description: Medium cloud level.
     highClouds:
       description: High cloud level.
+latitude:
+  description: Manually specify latitude.
+  required: false
+  type: number
```
Comment (klaasnicolaas) [inline source/_components/sensor.yr.markdown]: Is number a legit type? Or should it be a float?

Comment (Danielhiversen) [reply source/_components/sensor.yr.markdown]: Is not a float a sub type of number?

Comment (klaasnicolaas) [reply source/_components/sensor.yr.markdown]: Well I see [here](https://developers.home-assistant.io/docs/en/documentation_create_page.html) number not as variable type.

Comment (frenck) [reply source/_components/sensor.yr.markdown]: In the docs that would be translated into a `float`.
`number` is not valid in the docs.

**Review by frenck — APPROVED** [review]

**Pull request merged**